spec.bs

```diff
@@ -352,6 +356,9 @@ A user agent holds a <dfn>contribution cache</dfn>, which is a [=list=] of
 A user agent holds a <dfn>debug scope map</dfn>, which is an [=ordered map=]
 from [=debug scopes=] to [=debug details=].
 
+A user agent holds a <dfn>developer mode enabled</dfn>, which is a [=boolean=],
```

Comment:
Shall we align with ARA and use "The user-agent holds an associated boolean <dfn>automation local testing mode</dfn> (default false)."

Reply:
I removed associated to align with the other storage language, but moved it to the user-agent automation section to align with ARA